Playground/Plugins/Services/MockAuthentication/MockAuthentication.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<!--
+********************************************************************************************************
+
+Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License").
+You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+Unless required by applicable law or agreed to in writing, software distributed under the License
+is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and limitations under the License.
+
+********************************************************************************************************
+-->
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <Target Name="PostBuild" AfterTargets="PostBuildEvent">
```

Comment:
Can we add here (and each cases we do that) an extra check and breaking the build if something is wrong? e.g. does $(SolutionDir)\bin\$(Configuration)\net6.0\Plugins\Services\ exist?

Reply:
We can probably add some checks, but currently, if the folder does not exist, it would get created by the command. Also can we do it in a separate Jira - small issue but might take a bit of time.

Reply:
As I stated before I'll open another jira for testing that the path exists, but for now I'll resolve this issue.